mm2src/common/common.rs

```diff
@@ -1711,3 +1694,47 @@ impl<Id> PagingOptionsEnum<Id> {
 impl<Id> Default for PagingOptionsEnum<Id> {
     fn default() -> Self { PagingOptionsEnum::PageNumber(NonZeroUsize::new(1).expect("1 > 0")) }
 }
+
+#[cfg(target_arch = "wasm32")]
+pub fn register_wasm_log() {
```

Comment:
I think this can be moved into `common::log::wasm_log`. It'll allow crates like `mm2_db`, `mm2_net` not to depend on `common`

Reply:
Isn't log is also part of `common` crate? So we will still need to use `common` for`mm2_db` and `mm2_net`. Am I missing something ? :)

But I move it into `wasm_log` module anyway since it fits better there.

Reply:
I haven't checked, but it would be great to separate the `log` module into a `mm2_log` crate later. I hope there are no cross dependencies

Reply:
Yeah, seperating the log specific modules into new crate as `mm2_log` is already in my todo list.